### PR TITLE
Merge "shooter-development" to "development" 🦓 

### PR DESCRIPTION
**Pull request by yetiDS**
Merge the "shooter-development" branch to the main branch "development" 👍 
image

### CODE REVIEW

File: src/main/java/frc/robot/commands/IndexCommand.java
Comment (Amehta996) [inline src/main/java/frc/robot/commands/IndexCommand.java:1]: This command isn't tested, changes to intake mechanism might alter.

**Review by Amehta996 — APPROVED** [review]
Shooting works, good to merge

**Review by prthik — COMMENTED** [review]
approved

**Comment by samperlmutter** [comment]
@Amehta996 says there is nothing on this branch that shooterstate-development doesn't have

**Pull request closed without merging**